edge-agent/src/Microsoft.Azure.Devices.Edge.Agent.Kubernetes/edgedeployment/deployment/KubernetesDeploymentMapper.cs

```diff
@@ -209,6 +226,20 @@ List<V1EnvVar> CollectModuleEnv(KubernetesModule module, IModuleIdentity identit
                 envList.Add(new V1EnvVar(KubernetesConstants.EdgeK8sObjectOwnerKindKey, module.Owner.Kind));
                 envList.Add(new V1EnvVar(KubernetesConstants.EdgeK8sObjectOwnerNameKey, module.Owner.Name));
                 envList.Add(new V1EnvVar(KubernetesConstants.EdgeK8sObjectOwnerUidKey, module.Owner.Uid));
+                envList.Add(new V1EnvVar(KubernetesConstants.PortMappingServiceType, this.defaultServiceType.ToString()));
+                envList.Add(new V1EnvVar(KubernetesConstants.EnableK8sServiceCallTracingName, this.enableServiceCallTracing.ToString()));
+                if (this.persistentVolumeName.HasValue)
+                {
+                    envList.Add(new V1EnvVar(KubernetesConstants.PersistentVolumeNameKey, this.persistentVolumeName.OrDefault().ToString()));
+                }
+
+                if (this.storageClassName.HasValue)
+                {
+                    envList.Add(new V1EnvVar(KubernetesConstants.StorageClassNameKey, this.storageClassName.OrDefault().ToString()));
+                }
+
+                envList.Add(new V1EnvVar(KubernetesConstants.PersistentVolumeClaimDefaultSizeInMbKey, this.persistentVolumeClaimDefaultSizeMb.ToString()));
```

Comment:
persistentVolumeClaimDefaultSizeMb is optional right? Shouldn't we make it Option<uint> instead and not pass default(uint) everywhere?